addons/arrays/fnc_sortNestedArray.sqf

```diff
@@ -5,75 +5,43 @@ Description:
     Used to sort a nested array from lowest to highest using quick sort.
 
     Sorting is based on the specified column, which must have numerical data.
+    Original array is modified.
 
 Parameters:
-    _array: array - Nested array to be sorted
-    _index: integer - sub array item index to be sorted on
+    _array - Nested array to be sorted <ARRAY>
+    _index - Sub array item index to be sorted on <NUMBER>
+    _order - true: ascending, false: descending (optional: true) <BOOLEAN>

```

Comment:
Should the "optional: true" comment mean that 1) it is optional and 2) the default is "true"? In that case,
`(optional, default: true)` would be easier to understand.